src/containers/CheckoutPage.less

```diff
@@ -0,0 +1,364 @@
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+ @import './../style/common.less';
+ 
+.checkout-container {
+  position: relative;
+
+  .checkout-container-inner {
+    margin-left: -15px;
+    margin-right: -15px;
+
+    .view-title {
+      color: #000000;
+      text-transform: capitalize;
+
+      @media screen and (max-width: @mobileWidth - 1px) {
+        text-align: center;
+      }
+    }
+
+    .checkout-new-address-btn,
+    .checkout-edit-address-btn,
+    .checkout-delete-address-btn,
+    .checkout-new-payment-btn
+    {
+      border-radius: 4;
+    }
+
+    .checkout-title-container {
+      h1 {
+        position: relative;
+        float: left;
+        width: 100%;
+        min-height: 1px;
+        padding-left: 15px;
+        padding-right: 15px;
+        margin-top: 64px;
+        margin-bottom: 64px;
+      }
+    }
+
+    @media screen and (max-width: @mobileWidth - 1px) {
+      font-size: 16px;
+    }
+
+    .checkout-sidebar {
+      float: left;
+      width: 100%;
+      position: relative;
+      min-height: 1px;
+      padding-left: 15px;
+      padding-right: 15px;
+
+      @media (min-width: 992px) {
+        float: left;
+        width: 33.33333333333333%;
+        text-align: right;
+      }
+
+      @media (min-width: 1200px) {
+        float: left;
+        width: 25%;
+      }
+
+      .checkout-sidebar-inner {
+        display: inline-block;
+        padding: 32px;
+        border: 3px solid #040060;
+        border-radius: 6px;
+        background-color: rgba(255, 255, 255, 0.6);
+        text-align: left;
+        position: relative;
+
+        .btn-cmd-submit-order {
+          display: inline-block;
+          padding: 6px 12px;
+          margin-bottom: 0;
+          font-size: 14px;
+          font-weight: normal;
+          line-height: 1.428571429;
+          text-align: center;
+          vertical-align: middle;
+          cursor: pointer;
+          border: 1px solid transparent;
+          border-radius: 4px;
+          white-space: nowrap;
+          -webkit-user-select: none;
+          -moz-user-select: none;
+          -ms-user-select: none;
+          -o-user-select: none;
+          user-select: none;
+          border-radius: 2px;
+          -webkit-box-shadow: 0px 2px 0px 0px rgba(0, 0, 0, 0.2);
+          box-shadow: 0px 2px 0px 0px rgba(0, 0, 0, 0.2);
```

Comment:
Box shadow with 0px radius? This probably doesn't make any visual difference and you can probably remove it.